connector/servicegraphconnector/metadata.yaml

```diff
@@ -11,3 +11,28 @@ status:
 
 tests:
   config:
+
+telemetry:
+  metrics:
+    connector_servicegraph_dropped_spans:
```

Comment:
Shouldn't this be a breaking change? Before these metrics are `connector\servicegraph\dropped_spans` ect.

Reply:
They were only exported using Prometheus, which would convert those slashes into underscores.

Reply:
That's right, the only way this would be breaking is if someone is using the OTLP exporter, which is still very much experimental